x-pack/plugins/security_solution/common/detection_engine/rule_response_actions/schemas/endpoint.ts

```diff
@@ -6,14 +6,16 @@
  */
 
 import * as t from 'io-ts';
-import { RESPONSE_ACTION_API_COMMANDS_NAMES } from '../../../endpoint/service/response_actions/constants';
+import { ENABLED_AUTOMATED_RESPONSE_ACTION_COMMANDS } from '../../../endpoint/service/response_actions/constants';
 
 // to enable using RESPONSE_ACTION_API_COMMANDS_NAMES as a type
 function keyObject<T extends readonly string[]>(arr: T): { [K in T[number]]: null } {
   return Object.fromEntries(arr.map((v) => [v, null])) as never;
 }
 
 export const EndpointParams = t.type({
-  command: t.keyof(keyObject(RESPONSE_ACTION_API_COMMANDS_NAMES)),
+  command: t.keyof(keyObject(ENABLED_AUTOMATED_RESPONSE_ACTION_COMMANDS)),
```

Comment:
Hey, that's a great question. What we agreed on within the team is that we want to support just `isolate` command so far. This change, makes sure we get a schema error when user wants to force using another command. I am saying force, because in the UI it's limited to just that one command.